src/pages/ResourcePresentation.tsx

```diff
@@ -120,12 +121,20 @@ const ResourcePresentation: FC<ResourcePresentationProps> = ({ resource }) => {
           </StyledFeatureWrapper>
         )}
 
-        {resource.licenses && resource.licenses.length !== 0 && (
+        {resource.licenses && resource.licenses.length !== 0 && resource.licenses[0].identifier.length > 0 && (
           <StyledFeatureWrapper>
             <StyledCaption variant="caption">{t('resource.metadata.license')}</StyledCaption>
-            {resource.licenses.map((license) => (
-              <LicenseCard key={license.identifier} license={license} />
-            ))}
+            {resource.licenses.map((license) => {
+              if (license.identifier) {
+                return (
+                  <Card key={license.identifier}>
+                    <LicenseCard license={license} />
+                  </Card>
+                );
+              } else {
```

Comment:
trengs alternativ return ?   Kan gjerne forkorte ved å droppe return, og bytte ut if

Reply:
Array.prototype.map() expects a value to be returned at the end of arrow function  array-callback-return er feilmeldinga man får når den ikke returnerer noe

Reply:
Og hvis jeg bare hadde at den returnerte Card, så får man en grå stripe, som indikerer at licensetabellen har et element i seg som må være noe sånt som emtpyLicense eller noe.

Reply:
Det kan refaktoreres til dette:
`{resource.licenses.map((license) => (
              <>
                {license.identifier && (
                  <Card key={license.identifier}>
                    <LicenseCard license={license} />
                  </Card>
                )}
              </>
            ))}`